pytripgui/plan_executor/simulation_results.py

```diff
@@ -1,61 +1,57 @@
-import os
 import logging
 
-import pytrip as pt
-from pytrip import volhist
+from pytrip import volhist, DosCube, LETCube
 
 logger = logging.getLogger(__name__)
 
 
 class SimulationResults:
     def __init__(self, patient, plan):
         self.patient = patient
-        self.name = ""
-        self.plan = None
-        self.dose = None
-        self.let = None
+        self.name = plan.basename
+        self.plan = plan
```

Comment:
why do you have `self.name` variable if that name is stored under `self.plan.basename'?

Reply:
it's not my idea. It is also used in `__str__` function as name of simulation

Reply:
i think it will be better if plan and name are passed to contructor as parameters, then Demeter will be satisfied as well as her Law